### ISSUE TITLE
Enable XGP PDP-11/20

### ISSUE THREAD

**Issue by larsbrinkhoff** [PDP-10/its#1104]
This is to make use of the XGP-11.

Turn on TEN11P in ITS if that's not done already.  T11CPA must be set too.  Fix bugs #1080 and #1094.

Then set XGP to 1.  Run `:STUFF XGP` to load the PDP-11 software.  (Added in #926, built in #1077.)

**Comment by larsbrinkhoff** [comment]
@atsampson, have you seen any details about the XGP interface data format?  INFO; XGP 24 has a good section at the end of the SCAN node, but it ends with an incomplete sentence and a comma.

In the PDP-11 code, this is the data for a blank scan line:

    BNULL:  .BYTE 2,0,0,1           ;NULL SCAN LINE

If the bytes are to be swapped and this is a 0, 2 command then there is supposed to be bitmap data to follow, but there isn't.  Or the initial 2 could mean something else, but in that case it's not described what it is.

**Comment by larsbrinkhoff** [comment]
We now have SCRIMP (#860), which is the program that sends data to the XGP-11.

**Comment by larsbrinkhoff** [comment]
For the hardware simulation, read the 1973 file PJ; XGPHRD DOC

```
                        11-X:  PDP-11 TO XGP INTERFACE
  
                        HARDWARE DESCRIPTION

**Comment by larsbrinkhoff** [comment]
From MCLENN; ? 19

```
to restart xgp:
        sw = 0  laddr; dep
        sw = 1  laddr; dep; (or start?)
$~x. xgp $j
$~x. xgp2$j
$$~x.
stufer~k
        load from:
sysbin; vxgp bin
        to 11#:
1
        .... ok?
y
        ....*
xgp~k

or just try
halt/load/start
~z
:xgp

**Comment by larsbrinkhoff** [comment]
Fixed by #2271
